File: Electronics_Shop_17.Services/Database/ProductDiscount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Services.Database
{
    public class ProductDiscount : IHelperId
    {
        public int Id { get; set; }
        public int ProductId { get; set; }
        public int DiscountId { get; set; }
    }
}

File: angular17_web/src/app/models/product-discount/product.discount.ts
export class ProductDiscount {
  id!: number;
  productId!: number;
  discountId!: number;

  constructor(data?: any) {
    if (!data) return;

    this.id = data.id;
    this.productId = data.productId;
    this.discountId = data.discountId;
  }
}
